### DISCUSSION TITLE
SSV Token support in Ledger Live

### DISCUSSION THREAD

**Discussion by BenAffleck** [LedgerHQ/ledger-live-desktop#4312] [Coin/Token request]
Please add **SSV Token** support in Ledger Live.

The SSV contract is fully ERC20 compatible.

**SSV deployed address:** [0x9D65fF81a3c488d585bBfb0Bfe3c7707c7917f54](https://etherscan.io/token/0x9d65ff81a3c488d585bbfb0bfe3c7707c7917f54)
**Symbol:** SSV
**Decimals:** 18

**Rational**

SSV is part of the roadmap for ETH2.0, as provided by Vitalik Buterin and a critical piece to improve validator resilience and diversification on the eth2 proof-of-stake network. See [MPC Validators](https://twitter.com/vitalikbuterin/status/1333922620857745408) on Vitalik's tweet.

SSV stands for Secret-Shared-Validator and is a protocol for distributing the operations of an Ethereum validator between trustless parties using threshold signatures and a BFT (consensus) protocol. 

The SSV token replaces the [CDT Token](https://coinmarketcap.com/currencies/blox/), which is supported by Ledger Live. 
Holders of CDT can convert their Tokens to SSV. 

The SSV project is backed by prominent companies, such as the Ethereum Foundation, [Blox Staking](https://www.bloxstaking.com), Coinbase, Digital Currency Group, Lukka, etc.

**Additional reads**

- [Staking community grantee announcement](https://blog.ethereum.org/2021/02/09/esp-staking-community-grantee-announcement/)
- [An Introduction to Secret Shared Validators (SSV) for Ethereum 2.0](https://medium.com/bloxstaking/an-introduction-to-secret-shared-validators-ssv-for-ethereum-2-0-faf49efcabee)
- [Secret Shared Validators on Eth2](https://medium.com/coinmonks/eth2-secret-shared-validators-85824df8cbc0)
- [SSV.network Official Documentation](https://docs.ssv.network/)
- [Blox Receives Ethereum Foundation Grant to Develop Secret Shared Validator Nodes for Ethereum 2.0](https://www.bloxstaking.com/blog/product-updates/blox-receives-ethereum-foundation-grant-to-develop-secret-shared-validator-nodes-for-ethereum-2-0/)

**Comment by BenAffleck** [comment]
Binance announced SSV support today:
[https://www.binance.com/en/support/announcement/96e8866829784919a739c1855d21ef76](https://www.binance.com/en/support/announcement/96e8866829784919a739c1855d21ef76)

**Comment by BenAffleck** [comment]
Gate.io also announced support for SSV:
[https://www.gate.io/de/article/23091](https://www.gate.io/de/article/23091)

**Comment by BenAffleck** [comment]
SSV Network is now available on Coinmarketcap.com:
https://coinmarketcap.com/currencies/ssv-network/

**Comment by BenAffleck** [comment]
The Ledger support contacted me and confirmed the token to be listed soon:

> "We have added your token for the listing to be added in Ledger Live in future releases. "
> "Normally, if the token was checked and added for listing, it may be available in the next-after next release of Ledger Live."

It turned out that there is actually a simpler way to request a token listing:
https://developers.ledger.com/docs/token/erc20-bep20/

**Comment by JoeCoinz** [comment]
Please add support for Saitama Inu
https://etherscan.io/token/0x8b3192f5eebd8579568a2ed41e6feb402f93f73f
0x8b3192f5eebd8579568a2ed41e6feb402f93f73f
https://coinmarketcap.com/currencies/saitama-inu/
https://saitamatoken.com/

**Reply by BenAffleck** [reply]
Please open your own discussion. This is for SSV Token only. If you can, please remove your post. 
As to your request: You can add the token here: [https://developers.ledger.com/docs/token/erc20-bep20/](https://developers.ledger.com/docs/token/erc20-bep20/). Good luck.

**Comment by BenAffleck** [comment]
SSV Token support is now available in the Ledger Live Desktop app. Make sure to upgrade to the latest version.
